src/data/types/types.jsx

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/prefer-default-export */
+export const CHANGE_COUNT = 'CHANGE_COUNT';
+
```

Comment:
types/types.js is a little redundant and perhaps confusing.  Could we think of a better file name?